quick_intro_pt1.rst

```diff
@@ -634,14 +643,15 @@ and ``Basket`` class looking like this:
         public function addProduct($product)
         {
             $this->products[] = $product;
-            $this->productsPrice += $this->shelf->getProductPrice($product);
+            $this->productsPrice += $this->shelf
+                ->getProductPrice($product);
```

Comment:
I'm not a fan of this formatting. Having the method call on the next line is less readable IMO.